test/common/http/http1/codec_impl_test.cc

```diff
@@ -31,6 +33,7 @@ namespace Http1 {
 class Http1ServerConnectionImplTest : public TestBase {
 public:
   void initialize() {
+    http_parser_set_max_header_size(max_request_headers_kb_ * 1024);
```

Comment:
Would it make sense to defer this test change to a PR when we can set this via some codec interface?  It seems strange to test the limits of http_parser if we can't actually set it in our codec.

Reply:
+1

Reply:
Instead, I was able to bump the configurable limit up so this change makes sense again.